### ISSUE TITLE
reconnaissances territoriales

### ISSUE THREAD

**Issue by IkoDulac** [mouvement-16aout/Maison-democratie-Rimouski#8]
@jvtrudel, quand tu auras le temps est-ce que tu peux nous mettre ça en pied de page ou quelque part :   "Nous tenons à souligner que nous nous trouvons en territoire Mi’kmaqs et des Wolastoqiyiks (Malécites) non cédé."

**Comment by IkoDulac** [comment]
on pourra aussi se définir comme "safespace"...

**Comment by jvtrudel** [comment]
Intéressant. On en discutera la prochaine fois qu'on se verra. Est-ce qu'il y a des communautés Mi’kmaqs et/ou Wolastoqiyiks dans le coin?

**Comment by jvtrudel** [comment]
On pourrait peut-être s'inspirer de leur culture moins autoritaire et centralisatrice pour notre mouvement.

**Comment by IkoDulac** [comment]
rien de proche de Rimouski, il y a Cacouna et Pohénégamouk, puis Listugujet  Gesgapegiag dans la baie des chaleurs.
pour ce qui est de s'inspirer de leur culture, ce n'était pas le but des reconnaissances territoriales, et j'ai encore beaucoup à apprendre pour pouvoir le faire, mais pourquoi pas !

**Comment by IkoDulac** [comment]
mais je t'avoue que mon intention était plus modeste. juste la reconnaissance... pour l'instant !

**Comment by jvtrudel** [comment]
Oui, je comprends. Mais il ne faudrait pas se mettre les communautés autochtones à dos. Il parait que ça les gossent souvent ce genre de propos de la part des "blanc".

**Comment by IkoDulac** [comment]
?! t'as vu/lu quelque part un autochtone qui disait que la formule de reconnaissance territoriale (c'est bien de ça que tu parles quand tu dis: "ce genre de propos" ?) le gossait ?

**Comment by IkoDulac** [comment]
c'est la première proposition de Crystal Frase [ici](http://histoireengagee.ca/?p=7273)
il y a aussi un "guide des complices" qui en parle [là](http://www.indigenousaction.org/accomplices-not-allies-abolishing-the-ally-industrial-complex/)

"The starting point is to articulate your relationship to Indigenous Peoples whose lands you are occupying. This is beyond acknowledgment or recognition."
beyond acknowledgement... donc la reconnaissance reste un point de départ, un mini-minimum.

**Comment by jvtrudel** [comment]
Je suis pas contre. En fait, je suis 100% d'accord avec ta proposition. Mais est-ce le moment? Ça n'est pas l'objectif principal de notre groupe je crois (mais je peux me trompé). Point de vue stratégique, je propose qu'on fasse décoller le groupe. Ensuite, on proposera d'adopter quelques éléments des régimes politiques autochtones. Ensuite, le groupe sera plus ouvert à adopter ta position. Ton message (la reconnaissance de l'invasion des terres Malicite, etc.) aura bien plus de portée. On pourrait même faire des soirées de discussion sur le sujet.
